### ISSUE TITLE
The newest release lack of jxl related dependencies

### ISSUE THREAD

**Issue by hooke007** [shinchiro/mpv-winbuild-cmake#199]
https://github.com/mpv-player/mpv/commit/1345977f997611799c121dda9dee3870a6751421

**Comment by zhongfly** [comment]
`--enable-libjxl` needs to be enabled when compiling ffmpeg, and `libjxl` needs to be installed

**Comment by rorgoroth** [comment]
This is a bit of a PITA.
I'm not actually interested in jxl but I did try add it this morning in case it was a small simple affair that wouldn't cause me any future work.

So far I've added: gflags, brotli, highway and I can get libjxl to build outside of the prefix but when in the prefix as a package it just shits itself and wont configure complaining about brotli even though it says it found it.

Here is what I ended up with anyway: https://gist.github.com/rorgoroth/c14f89b5dc5fdcf6adc05d19e532a923

It should probably work dropped in to this project without much work and could be a starting point if anyone wants to get a proper working solution going assuming shin hasn't already got it figured out and hidden away somewhere 😅

But yeah, jxl and its deps (and all the bundled libs in release/git-submodules) seems quite a mess really.
